index.css

```diff
@@ -302,3 +305,79 @@
   align-items: left;
   font-weight: 700;
 }
+
+.contact_section {
+  padding: 150px 0 20px 0;
+  background-color: #6070ff;
+  border-top-left-radius: 150px;
+}
+
+.contact_form {
+  width: 90%;
+  margin: auto;
+}
+
+.contact_form header {
+  text-align: center;
+  color: #fff;
+  font-size: 40px;
+  font-weight: 700;
+}
+
+.contact_form p {
+  color: #ebebff;
+  font-size: 22px;
+  font-weight: 400;
+  line-height: 28px;
+  text-align: center;
+  padding: 15px 0;
+}
+
+fieldset {
+  border: none;
+  padding: 15px 0;
+  display: flex;
+  flex-direction: column;
+}
+
+label {
+  padding: 5px 0;
+}
+
+.input1 {
+  width: 100%;
+  color: #172b4d;
+  font-size: 17px;
+  font-weight: 400;
+  line-height: 28px;
+  padding: 15px 0;
+  outline: none;
+  border-radius: 10px;
+}
```

Comment:
- [ ] **[OPTIONAL]:** I would suggest adding some horizontal padding to your input fields 🤖. At the moment, whenever you type, there's no horizontal space:

![image](https://user-images.githubusercontent.com/77069011/199486062-8c526e0e-e8b6-4e2b-9d86-724ae6c4672f.png)